### DISCUSSION TITLE
How do turn off yanking on mouse selection?

### DISCUSSION THREAD

**Discussion by chtenb** [helix-editor/helix#12242] [Q&A]
Whenever I select some text I'm meeted with a message saying "yanked primary selection to register *".'
This means what I had in that register before is lost.
Is there a way to turn this behavior off? I would like to explicitly issue a yank command when I want it.

**Comment by kirawi** [comment]
Needs https://github.com/helix-editor/helix/pull/11414

**Reply by chtenb** [reply]
Right, I hadn't made that connection. So I could then just assign `m` or whatever, so the yanking doesn't get in the way (but still there when I need it)
